README.md

```diff
@@ -486,6 +486,11 @@ Next you can upload the required pieces of software:
     # Nothing is being flashed permanently yet.
     $ dfu-util --alt 3 -D emmc-boot-image-lxatac.fip
 
+    # Start a fastboot command in the background that disables the watchdog
+    # once the LXA TAC shows up as fastboot device.
+    # This also stops the automatic boot process.
+    $ fastboot oem exec "wd -x" &
```

Comment:
AFAIK, the watchdog is only enabled before booting Linux (`boot` command), so `wd -x` directly after DFU bring-up doesn't matter for watchdogs. As for stopping the automatic boot process, `fastboot flash` has the same effect, see:
https://github.com/barebox/barebox/blob/5423885658da4beddec619c1ec4b1cb741c68554/common/fastboot.c#L814

If this change fixes something, I suspect the issue is elsewhere.

Reply:
What I usually do is have an init script:
```
#!/bin/sh

if [ "$bootsource" = serial ]; then
   global.autoboot_timeout=30
fi
```

Reply:
Hmm. I would not want to change this back without re-investigating the `wd -x` change again.
But I would also prefer not to do such an investigation before we have our Gen 3 compatible release.
How about a compromise in the form of a draft PR #133 that does the suggested change and has a TODO item to re-investigate `wd -x` in the README?

Reply:
Ok.